pom.xml

```diff
@@ -433,6 +433,13 @@
                 <artifactId>org.eclipse.jgit</artifactId>
                 <version>4.3.1.201605051710-r</version>
             </dependency>
+
+            <!-- https://mvnrepository.com/artifact/com.google.guava/guava -->
+            <dependency>
+                <groupId>com.google.guava</groupId>
```

Comment:
What are we using this dependency for?

Reply:
This is the most reliable way I found to delete the NPM cache. Using `org.apache.commons.io.FileUtils` was throwing errors.